src/metabase/models/recent_views.clj

```diff
@@ -69,16 +72,30 @@
                 ;;       distinguish between them.
    :dashboard :table :collection])
 
-(defn- ids-to-prune-for-user+model [user-id model]
+(defn- moi->type
+  "Turns a model of interest into the base name of its table"
+  [moi]
+  (if (#{:model :card :metric} moi)
+    "card"
+    (name moi)))
+
+(mu/defn moi->model
+  "Given a model of interest, returns the model for it."
+  [moi :- (into [:enum] models-of-interest)]
+  (keyword "model" (str/capitalize (moi->type moi))))
```

Comment:
take it or leave it but IMO this would be clearer as just a map like
```
{:model      :model/Card
 :card       :model/Card
 :dashboard  :model/Dashboard
 :table      :model/Table
 :collection :model/Collection}
```
it's a small fixed set so this makes the mapping super easy to understand instead of needing to think about multiple different functions

Reply:
Thanks! fixed